docs/content/blog/new-logo.md

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
Is this an older/missed blog post?  Should it be intro'd in a different PR?

Reply:
oops, that's a draft I had on my computer. Sorry about that!